### PR TITLE
Added support for combining small_wood_fire_ext

### PR DESCRIPTION
**Pull request by Izzyb1234**
Combines the fule value of two small_wood_fire_ext blocks and replaces
it with a large_wood_fire_ext when you place one while pointing at
another.

error in naming branch.  combines small_wood_fire_ext for making charcoal

### CODE REVIEW

**Comment by Izzyb1234** [comment]
I removed support for combining thatch since it isn't working atm.  Combining snow slabs, unlit fire and charcoal slabs, extinguished fire & charcoal slabs.  Thatch uses stairs mod to generate the slabs which needs a different solution.  Will look at it again in a new PR at some point.

File: mods/minimal/utility.lua [hunk 1]
```diff
@@ -26,7 +26,7 @@ function minimal.safe_landing_spot(pos)
 				      def_top.walkable == true ) then
       return false -- loaded a solid node
    end
-   if dest_bot.name ~= "ignore" and ( def_bot and
+   if dest_bot.name ~= "ignore" and ( _bot and
```
Comment (jeremyshannon) [inline mods/minimal/utility.lua:29]: Looks like an accidental change here

**Review by jeremyshannon — COMMENTED** [review]
Looks good, apart from the change to the unrelated safe_landing_spot() function.

File: mods/minimal/utility.lua (hunk 1 above)
Comment (Izzyb1234) [reply mods/minimal/utility.lua:29]: hmm, not sure how that happened
I'll remove it

**Pull request merged**